### DISCUSSION TITLE
怎么设置某个国外IP直连

### DISCUSSION THREAD

**Discussion by loohalh** [vernesong/OpenClash#3718] [General]
<img width="1153" alt="image" src="https://github.com/vernesong/OpenClash/assets/16890588/5b7d6c18-6386-4349-b9a1-f7cedb780450">

我这样设置这个国外的iP直连，不经过openclash，但是不起作用 有大佬知道怎么弄吗

**Comment by ghost** [comment]
根据你的描述，不属于bug，就只回discuss这边了。没啥问题的话可以把issue关了

---

第二dns服务器和你这问题完全没关系，不用勾

---

你要的效果其实就是ip直连,或者说不想一个ip走代理流量

`- IP-CIDR,222.19.212.56/32,DIRECT`

![1](https://github.com/vernesong/OpenClash/assets/152784766/1b90a22a-e6eb-49f6-80ab-42a646076e6d)

**Comment by loohalh** [comment]
好的 在复写设置-规格设置-自定义规则里面添加了 可以了 谢谢&nbsp;


方寸
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年1月9日(星期二) 中午12:01
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [vernesong/OpenClash] 怎么设置某个国外IP直连 (Discussion #3718)





 
根据你的描述，不属于bug，就只回discuss这边了
 
 
第二dns服务器和你这问题完全没关系，不用勾
 
 
你要的效果其实就是ip直连,或者说不想一个ip走代理流量
 
- IP-CIDR,222.19.212.56/32,DIRECT
 
2024-01-09.120027.png (view on web)
